deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -6,7 +6,16 @@ const {
   encodePath,
   slugToFolder,
 } = require("@yari-internal/slug-utils");
+const { VALID_LOCALES } = require("@yari-internal/constants");
 
+const ONE_MONTH = 3600 * 24 * 30;
```

Comment:
Probably would call it something like `THIRTY_DAYS` since it's not a month in all cases